pkg/metrics/scheduler.go

```diff
@@ -84,8 +84,8 @@ func InitSchedulerMetrics() *SchedulerMetrics {
 			Help:      "Total number of attempts to allocate containers. State of the attempt includes `allocated`, `rejected`, `error`, `released`",
 		}, []string{"state"})
 
-	s.applicationSubmission = prometheus.NewCounterVec(
-		prometheus.CounterOpts{
+	s.applicationSubmission = prometheus.NewGaugeVec(
```

Comment:
Change to gauge because, we want to support dec also

Reply:
A submission can not be changed. When an application is submitted it will always be submitted. It should be a counter that keeps increasing. It is not a state that we track it is a pure counter from start to shutdown.

Reply:
Thanks @wilfred-s for clarify for this field,it makes sense to me.